src/lua/oltp_common.lua

```diff
@@ -416,14 +417,58 @@ function commit()
    stmt.commit:execute()
 end
 
+local ffi_enum_read = ffi.new('sb_counter_type', "SB_CNT_READ")
+local ffi_enum_drop = ffi.new('sb_counter_type', "SB_CNT_GUARANTEED_CAPACITY_DROP")
```

Comment:
I get an error on this line when I run warmup or prepare: 
`FATAL: ./oltp_common.lua:421: cannot convert 'string' to 'enum 173'`

Reply:
This is because the `sysbench` for prepare was using system sysbench while run was using custom sysbench.